src/include/txn.i

```diff
@@ -819,6 +840,10 @@ __wt_txn_read(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDATE *upd, WT
 
```

Comment:
We can skip reading history store for in-memory database.

Reply:
That's what `WT_CONN_HS_OPEN` is checking.